lib/bundle.js

```diff
@@ -228,7 +228,7 @@ function replaceSelectors(css, callback) {
   return matches
     .map((match, m) => {
       const val = normalizeSelector(match.value).replace('> > >', deepCombinator);
-      if (match.name === 'name' && val) {
+      if (match.name === 'name' && val && !val.trim().match(/^(\/\*)|(\/\/)/)) {
```

Comment:
@joeosburn can you extract out that last check as something like `notCommentedOut` or something?

Reply:
yep